src/gl/vbo_mesh.js

```diff
@@ -12,6 +12,7 @@ export default class VBOMesh  {
         this.gl = gl;
         this.vertex_data = vertex_data; // typed array
         this.vertex_layout = vertex_layout;
+        this.vertex_elements = options.vertex_elements;
         this.buffer = this.gl.createBuffer();
```

Comment:
Well `_buffer` (in my original implementation) was for the GL buffer itself (from GL terminology, e.g. `gl.createBuffer`). I was using `_data` as raw incoming ArrayBuffer data (so yes potential confusion over two "buffers"). In that nomenclature it would be `vertex_data` and `element_data` for the JS data, and `vertex_buffer` and `element_buffer` for the GL buffers. Or some other pairing would work, but I do think something that identifies these as distinct is good.